src/components/Pagination/Pagination.js

```diff
@@ -1,5 +1,5 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import React from 'preact-compat';
```

Comment:
While we are at it, maybe we could just `import PReact ...` no?

Reply:
It would imply some changes in the React component we wrote. The API of preact is a bit different.

Reply:
I meant to use `import PReact from 'preact-compat` but since there is a different API then I guess it's fine like that.

Reply:
you'd need to change the jsx calls too then (we can do that though)

Reply:
Probably not worth it I guess :)